src/standard_out_api.rs

```diff
@@ -1,95 +1,63 @@
+use crate::CommandLike;
+
```

Comment:
Do you have a rule for empty lines in `use` blocks? I typically keep all `use` statements without empty lines and let `rustfmt` sort them alphabetically. This part of the code isn't manually managed but auto-filled generated by my IDE.

Reply:
so I used to separated out use directives that come from this current crate and use directives from external crates/stdlib.

Nowadays I do that less I can combine this.

I still recommend the following though a newline between use for import and `pub use` for re-exported items. I like those two `use` blocks separate since they are maintained separately

Reply:
Makes sense, thanks!